charting/templates/charting/includes/chart-display.html

```diff
@@ -1,16 +1,16 @@
 <div class="chart-and-title">
-    {% if chart.title and chart.html_chart_titles %}
+  {% if chart.title and chart.html_chart_titles %}
```

Comment:
given we never use it is there a reason to check for `html_chart_titles`?

Reply:
removed.